app/helpers/application_helper.rb

```diff
@@ -123,4 +123,14 @@ def fullstory_script(current_user: nil)
       <script>#{base_script}</script>
     HTML
   end
+
+  # Copied from partner's application_helper.rb
+  # https://github.com/rubyforgood/partner/blob/f0df3bc0d3357461c358ab87eeceaa3d3e288ebf/app/helpers/application_helper.rb#L50-L55
+  # using helper_method and sticking it in base_controller doesn't work because of link_to
+  def sortable(column, title = nil)
+    title ||= column.titleize
+    css_class = column == sort_column ? "current #{sort_direction}" : nil
+    direction = column == sort_column && sort_direction == "asc" ? "desc" : "asc"
+    link_to title, { sort: column, direction: direction }, { class: css_class }
+  end
```

Comment:
Would it work if we added a helper file like:
```
app/helpers/partners/multi_item_form_helper.rb
```
I am not entirely sure but it looks like just defining the helper like in the file above, makes it's method available throughout the partner scope. Let's try that out :)

Reply:
TIL, looks like it works